README.md

```diff
@@ -1,4 +1,32 @@
-# codewind-odo-extension
-Extension to Codewind providing support for odo projects: https://codewind.dev
+# Codewind ODO extension
+Extension to Codewind providing support for ODO projects: https://codewind.dev
 
-README.md is in progress and will be completed soon.
+## Add additional rules to support Codewind ODO extension
+ODO extension needs add additional rules for accessing OpenShift resources
+Steps to add additional rules:
+1. In your home directory, run the following command to clone the ODO extension repository
+`git clone https://github.com/eclipse/codewind-odo-extension`
+2. Login to your OpenShift/OKD cluster
+3. Go into `~/codewind-odo-extension/odo-RBAC` then run the following commands to add additional rules
+`kubectl apply -f codewind-odoclusterrole.yaml`
+`kubectl apply -f codewind-odoclusterrolebinding.yaml`
+
+## Install Codewind ODO extension manually
+1. In your home directory, run the following command to clone the ODO extension repository
```

Comment:
In your home directory, run the following command to clone the ODO extension repository: